src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/Overview.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
```

Comment:
Maybe at least call these components GeneOverview, GeneExternalReverences?

Reply:
That's done